fs/zbd_zenfs.cc

```diff
@@ -437,6 +437,12 @@ void ZonedBlockDevice::LogGarbageInfo() {
       continue;
     }
 
+    // Need double check in case the zone become empty before we acquire
```

Comment:
The zone needs to be released before we continue. We should also be able to remove the empty check before Aquire(). One check should be enough?

Reply:
Yes, you are right, the first check could be removed.

I kept it there because I wanted to avoid zone acquiring as much as possible, but I am not sure how much optimization we can get.....If you think this optimize doesn't matter at all, I could remove it anyway